src/Pdp/PublicSuffixListManager.php

```diff
@@ -205,13 +284,16 @@ public function getList()
      */
     protected function write($filename, $data)
```

Comment:
The changes to `write()` seem like an awful lot of work to avoid using the `@` error control operator, and making changes to `error_reporting` in a third party library seems really invasive. I'd like to leave this method as it was, unless you have a really good reason for the changes.